traffic_ops/testing/api/v14/todb.go

```diff
@@ -275,10 +281,26 @@ INSERT INTO to_extension (name, version, info_url, isactive, script_file, server
 	return nil
 }
 
+// SetupServerCapabilities sets up seed server capabilities
+func SetupServerCapabilities(db *sql.DB) error {
```

Comment:
Just a reminder, this function should no longer be necessary after rebasing onto master, but you might want to just add the 'mem' and 'disk' capabilities to tc-fixtures.json now (currently has 'foo' and 'bar').